### PR TITLE
Performance References for FN Backends

### PR DESCRIPTION
**Pull request by fthaler**

### CODE REVIEW

**Comment by fthaler** [comment]
launch jenkins

**Comment by fthaler** [comment]
launch perftest

File: pyutils/perftest/references/daint_nvcc_cray/128.json [hunk 1]
```diff
@@ -5,8130 +5,9842 @@
...
+        0.000881427,
+        0.000863241
       ]
     },
     {
       "name": "horizontal_diffusion",
       "backend": "cpu_ifirst",
       "float_type": "float",
       "series": [
-        0.00048747,
```
Comment (havogt) [inline pyutils/perftest/references/daint_nvcc_cray/128.json:2255]: I think we should update the reference for this test again. In the perftest run that you did in the PR, we are back to the original (better numbers).

**Review by havogt — CHANGES_REQUESTED** [review]

**Comment by fthaler** [comment]
launch perftest

File: pyutils/perftest/references/daint_nvcc_cray/128.json (hunk 1 above)
Comment (fthaler) [reply pyutils/perftest/references/daint_nvcc_cray/128.json:2255]: Better now or not?

**Review by havogt — APPROVED** [review]
Looks better

**Pull request merged**